lib/stripe/ext/active_merchant/active_merchant.rb

```diff
@@ -37,6 +41,118 @@ def headers(options = {})
 
         headers
       end
+
+      # To create a charge on a card or a token, call
+      #
+      #   purchase(money, card_hash_or_token, { ... })
+      #
+      # To create a charge on a customer, call
+      #
+      #   purchase(money, nil, { :customer => id, ... })
+      def purchase(money, payment, options = {})
+        MultiResponse.run do |r|
+          if payment.is_a?(ApplePayPaymentToken)
+            r.process { tokenize_apple_pay_token(payment) }
+            payment = StripePaymentToken.new(r.params["token"]) if r.success?
+          end
+          r.process do
+            post = create_post_for_auth_or_purchase(money, payment, options)
+            commit(:post, 'charges', post, options)
+          end
+        end.responses.last
+      end
+
+      def store(payment, options = {})
+        params = {}
+        post = {}
+
+        if payment.is_a?(ApplePayPaymentToken)
+          token_exchange_response = tokenize_apple_pay_token(payment)
+          params = { card: token_exchange_response.params["token"]["id"] } if token_exchange_response.success?
```

Comment:
Ruby 2.0 syntax won't work by default, better to stick with the old syntax for now.

Reply:
Got it, easy fix